src/backend/assets/js/session.js

```diff
@@ -0,0 +1,66 @@
+$(document).ready(function() {
+  var sideslider = $('[data-toggle=collapse-side]');
+  var sel = sideslider.attr('data-target');
+  var top = $('header[role=banner]').outerHeight();
+  var downButton = $('#down-button');
+  var sessionElem = $('.single-session');
+  var sessionId = Number(sessionElem.attr('id'));
+  var maxNoOfSessions = 1000;
```

Comment:
This just assumes that the maximum number of sessions in an event is 1000 and creates an array of that size. It is used for the bookmark feature.